File: cs50/week6-Python/mario.py
from cs50 import get_int

while True:
    step = get_int("Height: ")
    if step < 9 and step > 0:
        break

for i in range(1, step + 1):
    print(' ' * (step - i) + '#' * i + '  ' + '#' * i)
    # for j in range(0, step - i):
    #     print(" ", end = '')

    # for k in range(0, i):
    #     print("#", end = '')

    # print("  ", end = '')

    # for l in range(0, i):
    #     print("#", end = '')

    # print()


    # for j in range(0, step - i):
    #     print(" ", end = '')

    # for k in range(0, i):
    #     print("#", end = '')

    # print("  ", end = '')

    # for l in range(0, i):
    #     print("#", end = '')

    # print()

File: cs50/week1-C/mario.c
#include <stdio.h>
#include "cs50.h"

int main(void)
{
	int steps = 0;
	int i, j, k;
	printf("enter the number of steps\n");
	steps = GetInt();
	while(steps > 23 || steps < 0)
	{
		printf("enter the number of steps\n");
		steps = GetInt();
	}

	printf("number of steps: %d\n", steps);
	for(i = 1;i <= steps;i++)
	{
		for(j = 1;j <= steps-i; j++)
		{
			printf(" ");
		}
		for(k = 1;k <= i+1; k++)
		{
			printf("#");
		}
		printf("\n");
	}
}

